web_client/src/store/index.ts

```diff
@@ -525,25 +527,25 @@ const {
     updateLastApiRequestTime(state) {
       state.lastApiRequestTime = Date.now();
     },
-    setLoadingFrame(state, value) {
-      state.loadingFrame = value;
+    setLoadingFrame(state, isLoading) {
+      state.loadingFrame = isLoading;
     },
-    setErrorLoadingFrame(state, value) {
-      state.errorLoadingFrame = value;
+    setErrorLoadingFrame(state, isErrorLoading) {
+      state.errorLoadingFrame = isErrorLoading;
     },
-    addScanFrames(state, { sid, id }) {
-      state.scanFrames[sid].push(id);
+    addScanFrames(state, { scanId, frameId }) {
+      state.scanFrames[scanId].push(frameId);
     },
-    addExperimentScans(state, { eid, sid }) {
-      state.scanFrames[sid] = [];
-      state.experimentScans[eid].push(sid);
+    addExperimentScans(state, { experimentId, scanId }) {
+      state.scanFrames[scanId] = []; // Why?
```

Comment:
I'm not sure why this was added here; if you take it out does behavior change?

Reply:
I'll check.